src/main/webapp/app/shared/layouts/main/main.component.ts

```diff
@@ -38,6 +44,16 @@ export class JhiMainComponent implements OnInit {
 
     ngOnInit() {
         this.router.events.subscribe((event) => {
+            if (event instanceof NavigationStart) {
+                const newShowSkeleton = this.shouldShowSkeleton(event.url);
+                if (!newShowSkeleton && this.showSkeleton) {
+                    this.renderer.addClass(this.document.body, 'transparent-background');
```

Comment:
Is there no other way to implement this? This looks quite ugly to be honest

Reply:
I researched this and to my understanding there is no other way to overwrite the default background attibute of the body tag as it is outside of Angular's reach. We have to manually set a class on the background for this.